apps/demo/src/app/cart/components/cart-totals/cart-totals.component.spec.ts

```diff
@@ -13,12 +13,17 @@ class WrapperComponent {
   @Input() cartValue: Cart;
 }
 
+@Pipe({name: 'currency'})
+class MockCurrencyPipe implements PipeTransform {
```

Comment:
@lderrickable as I'm sure we'll need to do this in several places, do you think it makes sense to migrate this into `@daffodil/core/testing`?

Reply:
Yeah, I think that makes sense. I added an angular-mocks folder in core, and I added a `mock-pipes.ts` file.